spectrum/checks.py

```diff
@@ -568,6 +568,15 @@ def just_load(self, path):
         _assert_status_code(response, 200, url)
         return response
 
+    def redirect(self, path, expected, status_code=301):
+        url = _build_url(path, self._host)
+        LOGGER.info("Loading %s", url)
+        response = requests.get(url, allow_redirects=False)
+        _assert_status_code(response, status_code, url)
+        location = response.headers['Location']
+        assert location.startswith(self._host)
```

Comment:
Bit redundant?

Reply:
It's for a clear failure if the scheme/host is wrong rather than the path